pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -1055,6 +1055,13 @@ func (pl *dynamicResources) lookupClassParameters(logger klog.Logger, class *res
 		}
 		return parameters, nil
 	default:
+		sort.Slice(objs, func(i, j int) bool {
+			obj1, obj2 := objs[i].(*resourcev1alpha2.ResourceClassParameters), objs[j].(*resourcev1alpha2.ResourceClassParameters)
+			if obj1 == nil || obj2 == nil {
+				return false
+			}
+			return obj1.CreationTimestamp.Before(&obj2.CreationTimestamp)
+		})
```

Comment:
should we sort it by name or creation timestamp?

Reply:
pull-kubernetes-linter-hints:

```
ERROR: pkg/scheduler/testing/wrappers.go:1182:34: ST1016: methods on the same type should have the same receiver name (seen 1x "p", 7x "wrapper") (stylecheck)
ERROR: func (p *ClaimParametersWrapper) CreationTimestamp(t metav1.Time) *ClaimParametersWrapper {
ERROR:                                  ^
ERROR: pkg/scheduler/testing/wrappers.go:1248:34: ST1016: methods on the same type should have the same receiver name (seen 1x "p", 6x "wrapper") (stylecheck)
ERROR: func (p *ClassParametersWrapper) CreationTimestamp(t metav1.Time) *ClassParametersWrapper {
ERROR:      
```

the function follows the podwrapper style. https://github.com/carlory/kubernetes/blob/f3cca34c90b2a9bae89b646ac0751ef9c2b01a18/pkg/scheduler/testing/wrappers.go#L291.

Reply:
Name. The error message doesn't include the time stamp, so the sort order would still look random to a user.

Thanks for following up in the bug report!

/lgtm
/approve

Reply:
Per offline discussion, it makes sense to keep the sorting in core logic.